android/res/layout/fragment_editor_host.xml

```diff
@@ -27,6 +28,7 @@
         tools:visibility="gone"/>
 
       <ImageView
+        app:tint="@color/image_view"
```

Comment:
android:tint should also work here, right?

Reply:
When I tried using android tint IDE was warning to use app:tint specifically.